README.md

```diff
@@ -17,3 +17,32 @@ looks like this:
     # Start Schema Registry
     docker run -d --name schema-registry --link zookeeper:zookeeper \
         --link kafka:kafka confluent/schema-registry
+
+    # Start Schema Registry
+    docker run -d --name schema-registry --link zookeeper:zookeeper \
+        --link kafka:kafka confluent/schema-registry
+
+
+## Running on Multiple Remote Hosts and Clustering
+To run across multiple hosts you will need some way of communicating between Docker hosts so all remote containers can see each other. This is typically done via some sort of service discovery mechanism (so containers/services can find each other) and/or SDN (so containers can communicate) such as [weave](http://weave.works/) or [flannel](https://github.com/coreos/flannel) as SDN examples. Having that in place, you can use environment variables to specify the IP/hostname and respective ports for the remote containers and forgo the use of `--link`. For example to make a 3-node Zookeeper ensemble, each running on separate Docker hosts (zk-1:172.16.42.101, zk-2:172.16.42.102, and zk-3:172.16.42.103), and have a remote Kafka 2-node cluster connection:
+
+```sh
+docker run --name zk-1 -e zk_id=1 -e zk_server.1=172.16.42.101:2888:3888 -e zk_server.2=172.16.42.102:2888:3888 -e zk_server.2=172.16.42.103:2888:3888 -p 2181:2181 -p 2888:2888 -p 3888:3888 confluent/zookeeper
```

Comment:
Looks like there's a typo here -- `zk_server.2` is repeated twice and `zk_server.3` is missing? Although the IP address was updated correctly. Same across all 3 ZK containers.